backend/app/api/src/endpoints/global/events/PatchEventsEndpoint.ts

```diff
@@ -144,44 +115,20 @@ export class PatchEventsEndpoint extends Endpoint<Params, Query, Body, ResponseB
                 })
             }
 
-            event.meta = patchObject(event.meta, patch.meta)
+            event.meta = patchObject(event.meta, patch.meta);
 
-            if (patch.organizationId !== undefined) {
-                if (organization?.id && patch.organizationId !== organization.id) {
-                    throw new SimpleError({
-                        code: 'invalid_data',
-                        message: 'Invalid organizationId',
-                        human: 'Je kan geen activiteiten aanmaken voor een andere organisatie',
-                    })
-                }
-    
-                if (!organization?.id && !Context.auth.hasPlatformFullAccess()) {
-                    throw new SimpleError({
-                        code: 'invalid_data',
-                        message: 'Invalid organizationId',
-                        human: 'Je kan geen activiteiten voor een specifieke organisatie aanmaken als je geen platform hoofdbeheerder bent',
-                    })
-                }
-    
-                const eventOrganization = patch.organizationId ? (await Organization.getByID(patch.organizationId)) : null
-                if (!eventOrganization && patch.organizationId) {
-                    throw new SimpleError({
-                        code: 'invalid_data',
-                        message: 'Invalid organizationId',
-                        human: 'De organisatie werd niet gevonden',
-                    })
-                }
+            if(patch.organizationId) {
                 event.organizationId = patch.organizationId
-                event.meta.organizationCache = eventOrganization ? NamedObject.create({id: eventOrganization.id, name: eventOrganization.name}) : null
-            } else {
-                // Update cache
-                if (event.organizationId) {
-                    const eventOrganization = await Organization.getByID(event.organizationId)
-                    if (eventOrganization) {
-                        event.meta.organizationCache = NamedObject.create({id: eventOrganization.id, name: eventOrganization.name})
-                    }
-                }
             }
+            
+            const eventOrganization = await this.tryAdminEvent(event, { organization: originalOrganization });
```

Comment:
De cache hier doorsturen is eigenlijk overbodig, want AdminPermissionChecker gaat de organization al intern hebben gecached, dus je kan de optie beter schrappen